app/gui/src/project-view/stores/suggestionDatabase/entry.ts

```diff
@@ -191,11 +191,15 @@ const DOCUMENTATION_ROOT = 'https://help.enso.org/docs/api'
 /** TODO: Add docs */
 export function suggestionDocumentationUrl(entry: SuggestionEntry): string | undefined {
   if (entry.kind !== SuggestionKind.Method && entry.kind !== SuggestionKind.Function) return
-  const { project, path } = entry.definitionPath
-  const definedPath = entry.definedIn.path + '.'
-  const usedPath = definedPath ? path?.replace(definedPath, definedPath?.replaceAll('.', '/')) : path
-  const concatenatedPath = `${DOCUMENTATION_ROOT}/${project}/${usedPath}`
-  return concatenatedPath
+
+  const { project, path } = entry.definedIn
+  if (!project?.startsWith('Standard.') || !path) return
+
+  const functionPath = entry.definitionPath.path
+  if (!functionPath) return
+  const postPath = functionPath.replace(`${path}.`, '')
+
+  return [DOCUMENTATION_ROOT, project, ...qnSegments(path), postPath].join('/')
```

Comment:
Some comment here would be useful, because I spent a while considering, why do we remove path only to bring it back with `..qnSegments(path)`